website/content/docs/api-clients/api.mdx

```diff
@@ -79,3 +79,169 @@ The following method conventions are used within Boundary's API:
 ### DELETE
 
 `DELETE` is used for deleting a specific resource, and is only used against a particular resource path.
+
+## Rate limiting
+
+If your controllers try to process every API request, it can lead to situations when the controllers are overwhelmed with requests.
+The controllers may either run out of resources or overwhelm the database server and exhaust its resources.
+API rate limiting lets you configure limits on the rate of API requests to help manage your resources and prevent them from being overwhelmed.
+
+Boundary creates quotas to track the number of requests in a given time period.
+You can configure Boundary to track the total number of requests or track requests by auth token or IP address.
+Boundary reserves a portion of its memory for tracking quotas to ensure that it does not consume too much memory if there is a sudden burst of requests.
+If Boundary is unable to store a quota, it limits the request with a 503 HTTP status code.
+
+API rate limiting is enforced on the controllers.
+By default, the `list` action has a limit of 120 requests per minute for all resources.
```

Comment:
I am going to be changing these default limits soon, I am still doing some testing, but I'll send you the updated numbers when I have them.